app/javascript/packs/PF4Styles/application.js

```diff
@@ -0,0 +1,8 @@
+import 'patternflyStyles/modal'
```

Comment:
I know we've been importing selectively, but do you think we may import the whole PF css file? This is growing too much

Reply:
Adding all PF css in the main layout would be ideal but right now it messes with lots of elements. I think even if it's growing a lot we have fine control over our styles.